verification/cbmc/proofs/run-cbmc-proofs.py

```diff
@@ -0,0 +1,258 @@
+#!/usr/bin/env python3
```

Comment:
This file needs different permissions. Please run `chmod u+x ./run-cbmc-proofs.py` and update your PR

Reply:
implemented as suggested.